packages/forms/resources/js/components/markdown-editor.js

```diff
@@ -92,6 +92,7 @@ export default function markdownEditorFormComponent({
     state,
     translations,
     toolbarButtons,
+    uploadFileAllowed,
```

Comment:
Please rename this to canAttachFiles and reorder it so its still in alphabetical order in the arg lists

Reply:
Hello! Done.